exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterMetrics.java

```diff
@@ -70,7 +70,7 @@ public void addFailed(long value) {
 
   private LongCounter seen() {
     LongCounter seen = this.seen;
-    if (seen == null) {
+    if (seen == null || isNoop(seen)) {
```

Comment:
The upside of this is that it provides a simple way to get around the race of `ExporterMetrics` possibly initializing before the `Supplier<MeterProvider>` is able to resolve a concrete `SdkMeterProvider` instance.

The downside is:
- Doing extra work every time we record a measurement
- When a noop is installed, we'll be doing extra work to call `meter().counterBuilder(exporterName + ".exporter.seen").build()` on every export for the lifetime of the app

Still, I think the benefits are worthwhile since in the grand scheme of things, the extra work being done is a drop in the bucket compared to the work to send batches of telemetry over the network.

Reply:
For cases where a valid provider is never supplied, yes it will always result in a bit of extra work, but it will generally be on a background thread, so of minor consequence.
Otherwise, the extra work is only until a valid instance is returned since the null check will short circuit the check.